src/parser/deathknight/unholy/modules/features/VirulentPlagueEfficiency.tsx

```diff
@@ -63,12 +64,12 @@ class VirulentPlagueEfficiency extends Analyzer {
         average: (3),
         major: (5),
```

Comment:
why the () around the numbers here?

Reply:
/shrug

This code wasn't looked at for almost all of BFA, who knows what Biceps and I were thinking back in Legion